.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Telegram message
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    
+jobs:
+  build:
+    name: Send Telegram message
+    runs-on: ubuntu-latest
+    steps:
+      - name: send telegram message on push
+        uses: appleboy/telegram-action@master
+        with:
+          to: ${{ secrets.TELEGRAM_TO }}
+          token: ${{ secrets.TELEGRAM_TOKEN }}
+          format: markdown
+          message: |
+            👨‍💻 *New PR Updates from ${{ github.actor }}*
+            
+            ⭐️ *Latest commit*: ${{ github.event.commits[0].message }}
+            
+            🚀 *PR*: https://github.com/${{ github.repository }}
+            
```

Comment:
Should there be a link to PR and not to repository?

Reply:
Nice catch. Just updated